arch/arm/boot/dts/bcm283x.dtsi

```diff
@@ -565,6 +565,34 @@
 			status = "disabled";
 		};
 
+		csi0: csi0@7e800000 {
```

Comment:
The node name should describe the function like csi, so my suggestion would be:
csi0: csi@7e800000

Reply:
Done